vllm/distributed/eplb/eplb_state.py

```diff
@@ -160,7 +160,7 @@ class EplbState:
 
     buffer_lock: threading.Lock = threading.Lock()
 
-    expert_buffer: Optional[list[torch.Tensor]] = None
+    expert_buffer:list[torch.Tensor] = []
```

Comment:
Mutable default is not safe for builtin `dataclass` (it is safe for pydandic dataclasses though if you wanted to switch to those)

```suggestion
    expert_buffer: list[torch.Tensor] = field(default_factory=list)
```